src/main/java/org/dataloader/stats/Statistics.java

```diff
@@ -0,0 +1,79 @@
+package org.dataloader.stats;
+
+import java.util.Map;
+
+/**
+ * A simple set of statistics about {@link org.dataloader.DataLoader} operations
+ */
+public interface Statistics {
```

Comment:
Not sure an interface is really needed here.

Reply:
done